applications/RANSModellingApplication/CMakeLists.txt

```diff
@@ -0,0 +1,89 @@
+set(CMAKE_INCLUDE_CURRENT_DIR ON)
+
+message("**** configuring KratosRANSModellingApplication ****")
+
+# check for fluid dynamics application
+if(NOT (${FLUID_DYNAMICS_APPLICATION} MATCHES ON))
+  message(FATAL_ERROR "RANSModellingApplication requires FluidDynamicsApplication, Please re-compile with -DFLUID_DYNAMICS_APPLICATION=ON flag.")
+endif(NOT (${FLUID_DYNAMICS_APPLICATION} MATCHES ON))
+
+################### PYBIND11
+include(pybind11Tools)
+
+include_directories( ${CMAKE_SOURCE_DIR}/kratos )
+include_directories( ${CMAKE_SOURCE_DIR}/applications/FluidDynamicsApplication )
+
+# generate variables with the sources
+set( KRATOS_RANS_MODELLING_APPLICATION_CORE
```

Comment:
can you please use the glob-recurse for new apps? See Structural
=> please check regarding MPI to not include too much